takwimu/profiles/census.py

```diff
@@ -43,6 +43,30 @@
                     'title': 'WorldBank, 2018',
                 },
             },
+            'households_and_population': {
+                'source': {
+                    'link': 'https://www.knbs.or.ke/?wpdmpro=2019-kenya-population-and-housing-census-volume-i-population-by-county-and-sub-county',
+                    'title': '2019 Kenya Population and Housing Census Volume I',
+                },
+            },
+            'avg_household_size': {
+                'source': {
+                    'link': 'https://www.knbs.or.ke/?wpdmpro=2019-kenya-population-and-housing-census-volume-i-population-by-county-and-sub-county',
+                    'title': '2019 Kenya Population and Housing Census Volume I',
+                },
+            },
+             'population_sex_2019': {
+                'source': {
+                    'link': 'https://www.knbs.or.ke/?wpdmpro=2019-kenya-population-and-housing-census-volume-i-population-by-county-and-sub-county',
+                    'title': '2019 Kenya Population and Housing Census Volume I',
+                },
+            },
+             'subcounty_population_sex_2019': {
```

Comment:
1. We don't have subcounties in Takwimu/HURUmap just yet @KhadijaMahanga, correct?
2. Any reason why we're keeping these in a separate table and not the `population_sex_2019`?

Reply:
we don't have subcounty that's the reason they're in different tables... so `subcounty population sex 2019` table has subcounty as a field. Meaning its population for county per each subcounty per gender.

Reply:
@kilemensi

Reply:
Would it break anything if the data is in the same table @KhadijaMahanga? Or can we hold it until we add sub-counties?

Reply:
Ahh... I see we're using it as just _data_ at county level.
![Population per subcounty 2019](https://user-images.githubusercontent.com/1779590/69852653-e3d47480-1295-11ea-9ced-ba346a99ca47.png)

:rocket: @KhadijaMahanga

Reply:
They're called "constituencies". 🙄